### PR TITLE
Add test for CreateLanguage_CreatesLanguage

### PR DESCRIPTION
**Pull request by Simply007**
### Motivation

Related to #69 

Test out the new approach of Unit tests that are not dependent on real Kontent Project.

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

**Comment by Simply007** [comment]
I would use the same HTTP bodies we have in our swagger examples:
[MAPI Swagger definition](https://kcdmaster.blob.core.windows.net/api-reference-specifications/management_api_v2.json)

File: Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/LanguagesTests.cs [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                Name = "German (Germany)",
+                Codename = "de-DE",
+                IsActive = false,
+                ExternalId = "standard-german",
+                FallbackLanguage = Reference.ById(Guid.Parse("00000000-0000-0000-0000-000000000000"))
+            };
+
+            var response = await client.CreateLanguageAsync(newLanguage);
+
+            Assert.Equal(newLanguage.Name, response.Name);
```
[the remark this answers is not in the document]
Comment (Simply007) [reply Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/LanguagesTests.cs]: 19617a7e

File: Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/LanguagesTests.cs [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            Assert.Equal(newLanguage.Codename, response.Codename);
+            Assert.Equal(newLanguage.ExternalId, response.ExternalId);
+            Assert.Equal(newLanguage.FallbackLanguage.Id, response.FallbackLanguage.Id);
+        }
+
+        [Fact]
+        public async void ListLanguages_ListsLanguages()
+        {
+            var client = _fileSystemFixture.CreateDefaultMockClientRespondingWithFilename("ListLanguages_ListsLanguages.json");
+
```
[the remark this answers is not in the document]
Comment (Simply007) [reply Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/LanguagesTests.cs:50]: e616b8b9

File: Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/FileSystemFixture.cs [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            _managementOptions = new ManagementOptions()
+            {
+                ApiKey = _configuration.GetValue<string>("ApiKey", "Dummy_API_key"),
+                ProjectId = _configuration.GetValue<string>("ProjectId", PROJECT_ID)
+            };
+            _urlBuilder = new EndpointUrlBuilder(_managementOptions);
+            _messageCreator = new MessageCreator(_managementOptions.ApiKey);
+        }
+
+        public ManagementClient CreateMockClient(IManagementHttpClient httpClient)
```
[the remark this answers is not in the document]
Comment (Simply007) [reply Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/FileSystemFixture.cs]: I would like to have it public for potential tests.

File: Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/FileSystemFixture.cs [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        {
+            var actionInvoker = new ActionInvoker(httpClient, _messageCreator);
+            return new ManagementClient(_urlBuilder, actionInvoker);
+        }
+
+        public ManagementClient CreateDefaultMockClientRespondingWithFilename(string responseFileName)
```
[the remark this answers is not in the document]
Comment (Simply007) [reply Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/FileSystemFixture.cs]: I have changed it to `CreateMockClientWithResponse` - af38ff4b

File: Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/FileSystemFixture.cs (hunk 3 above)
[the remark this answers is not in the document]
Comment (gormal) [reply Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/FileSystemFixture.cs]: alright

**Review by gormal — APPROVED** [review]
LGTM!

**Pull request merged**